frontend/src/Navigation/index.jsx

```diff
@@ -7,6 +7,7 @@ import Settings from '../Settings';
 import colors from '../common/colors';
 import Game from '../Game';
 import Login from '../Login';
+import AuthLoadingScreen from '../AuthLoadingScreen';
```

Comment:
we are going to use an animation for loading, here are some [demos](https://dribbble.com/search/loading).